application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -43,9 +45,11 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
             return;
         }
 
-        event
-            .replyEmbeds(MessageUtils.generateEmbed(null, tagSystem.getTag(id).orElseThrow(),
-                    event.getUser(), TagSystem.AMBIENT_COLOR))
-            .queue();
+        event.replyEmbeds(tagEmbed(id, event.getUser(), event.getCommandString())).queue();
+    }
+
+    private MessageEmbed tagEmbed(String id, User user, String commandString) {
+        return MessageUtils.generateEmbed(null, tagSystem.getTag(id).orElseThrow(), user,
+                TagSystem.AMBIENT_COLOR, user.getName() + " " + commandString);
```

Comment:
I think the footer reads a bit weird. maybe dont just add a space in between but a `used`, so `+ " used " + `